app/views/layouts/application.html.erb

```diff
@@ -4,11 +4,21 @@
     <title>Evensteven</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag '//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css' %>
```

Comment:
If you use font awesome, please add it using the gem. Serving assets through CDN might cause unforeseen behavior at times and is considered bad practice.